opentracing-spring-cloud-starter/pom.xml

```diff
@@ -16,5 +16,86 @@
       <groupId>${project.groupId}</groupId>
       <artifactId>opentracing-spring-cloud</artifactId>
     </dependency>
+
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.opentracing</groupId>
+      <artifactId>opentracing-mock</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-test</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-web</artifactId>
+      <scope>test</scope>
+    </dependency>
   </dependencies>
+
+  <build>
```

Comment:
Why are you excluding the tests from the compilation?

Reply:
It's a hack... without this it does not compile because required dependencies are is a separate profiles